.gitignore

```diff
@@ -1,4 +1,4 @@
 CMakeFiles
 Makefile
 *.a
-*.cmake
```

Comment:
This is a rule I generally put for the case of "in-source" builds, so that my build files aren't picked up if I lazily build in the same directory.